src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+}
+
+.calendar__day {
+  display: flex;
+  position: relative;
+  height: $day-size;
+  width: $day-size;
+  box-sizing: border-box;
+  background-color: $background-color;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  border: solid $border-size black;
+  justify-content: center;
+  align-items: center;
+  transition:
+    background-color $animation-duration,
+    transform $animation-duration;
+
+  &:hover {
+    cursor: pointer;
+    transform: translateY(-20px);
+    background-color: $hover-color;
+  }
+}
```

Comment:
The `.calendar__day` class has a lot of styles, which is fine, but it could be better to split it into smaller, reusable classes. For example, you could have a `.box` class for common styles like `display: flex`, `justify-content: center` and `align-items: center`. This would make your CSS more modular and easier to maintain.